lib/paperclip/thumbnail.rb

```diff
@@ -76,7 +77,11 @@ def make
 
         parameters = parameters.flatten.compact.join(" ").strip.squeeze(" ")
 
-        success = convert(parameters, :source => "#{File.expand_path(src.path)}#{'[0]' unless animated?}", :dest => File.expand_path(dst.path))
+        success = convert(parameters, 
+          :source => "#{File.expand_path(src.path)}#{'[' + @frame_index.to_s + ']' unless animated?}", 
```

Comment:
@tute one more thing, i looks like ANIMATED_FORMATS only includes gif right now, which may be why the animated? method was returning false for videos.  I can update that as well if we want to make this method work only when animated is true, which makes the most sense.